rig/machine_control/machine_controller.py

```diff
@@ -731,16 +731,22 @@ def _get_next_nn_id(self):
         self._nn_id = self._nn_id + 1 if self._nn_id < 126 else 1
         return self._nn_id * 2
 
-    def _send_ffs(self, pid, region, n_blocks, fr):
+    def _send_ffs(self, pid, n_blocks, fr):
         """Send a flood-fill start packet."""
         sfr = fr | (1 << 31)
         self._send_scp(
             0, 0, 0, SCPCommands.nearest_neighbour_packet,
             (NNCommands.flood_fill_start << 24) | (pid << 16) |
-            (n_blocks << 8),
-            region, sfr
+            (n_blocks << 8), 0x0, sfr
         )
 
+    def _send_ffcs(self, region, core_mask, fr):
+        """Send a flood-fill core select packet."""
```

Comment:
Might be good to say in brief:
* This packet type was added in a branch of scamp-134
* What the packet does (as ST is the only docs and he didn't invent this, he may forget...)